assets/javascripts/omise-myaccount-card-handler.js

```diff
@@ -142,5 +81,37 @@
 	$("#omise_add_new_card").click(function(event){
 		create_card();
 	});
+
+	function saveCard(payload) {
+		const data = {
+			action: "omise_create_card",
+			omise_token: payload.token,
+			omise_nonce: $("#omise_add_card_nonce").val()
+		};
+		$.post(omise_params.ajax_url, data, 
+			function(wp_response){
+				if(wp_response.id){
```

Comment:
Should we add a JavaScript linter? This file has different indentation/formatting than the others.

Reply:
Not sure we can add sinceWooCommerce do not use `npm`